src/main/scala/com/lightbend/kafkalagexporter/MetricsReporter.scala

```diff
@@ -24,7 +24,7 @@ object MetricsReporter {
     case (context, Stop(sender)) =>
       Behaviors.stopped { () =>
         metricsSink.stop()
-        context.log.info("Gracefully stopped Prometheus metrics endpoint HTTP server")
+        context.log.info("Gracefully stopped metrics sink")
```

Comment:
I think it would be useful here if each sink had a name so it could be logged.

Reply:
Each sink has a name assigned now and stored in their config object. However, this is not accessible within the scope of this class.

As this is larger change to refactor it, I propose we handle it as separate refactoring effort to log properly with sink names. What do you think ?

Reply:
Couldn't a `name` field be added to `MetricsSink` or `EndpointSink` and referenced here?

Reply:
This is addressed by displaying the ClassName that is equivalent to `name` field.  Can you please resolve this, if that's okay ?